src/core/server/http/lifecycle_handlers.ts

```diff
@@ -31,7 +31,11 @@ export const createXsrfPostAuthHandler = (config: HttpConfig): OnPostAuthHandler
   const { whitelist, disableProtection } = config.xsrf;
 
   return (request, response, toolkit) => {
-    if (disableProtection || whitelist.includes(request.route.path)) {
+    if (
+      disableProtection ||
```

Comment:
It seems that we added `server.xsrf.disableProtection` in https://github.com/spalger/kibana/pull/6 for testing purposes solely. I'm wondering if we want to collect all test-only API in the one document to have a better understanding of API surface in case of future refactoring. (in addition to `--migrationgs.skip`, `/internal/saved_objects/_migrate` etc)